docs/mining.md

```diff
@@ -21,9 +21,9 @@ certificate into the default system Certificate Authority list.
 
 ## Ubuntu
 
-1. Copy rpc.cert to /usr/share/ca-certificates: `# cp /home/user/.btcd/rpc.cert /usr/share/ca-certificates/btcd.crt`
-2. Add btcd.crt to /etc/ca-certificates.conf: `# echo btcd.crt >> /etc/ca-certificates.conf`
-3. Update the CA certificate list: `# update-ca-certificates`
+1. Copy rpc.cert to /usr/share/ca-certificates: `cp /home/user/.btcd/rpc.cert /usr/share/ca-certificates/btcd.crt`
+2. Add btcd.crt to /etc/ca-certificates.conf: `echo "btcd.crt" >> /etc/ca-certificates.conf`
```

Comment:
TL;DR;
my bad. thank for merging/reviewing the rest. 

TL;DR;
clear case of "just a habit"!
but honestly: I grepped through my `.bash_history` to find out what made my thinking that the quotes were needed. I have a blurry idea I saw an error popping up. But it was a typo on my side (`btccd.cert` instead of `btcd.cer`t) hence my `update-ca-certificates` failed. Half-sleepy already must have misinterpreted that.

Reply:
No worries.  I think so much of what we do with the shell comes down to habit and may or may not be necessary :)

Anyway, if you could remove the quotes I'll approve and merge.